packages/client/src/generation/generateClient.ts

```diff
@@ -325,6 +326,12 @@ export async function generateClient(options: GenerateClientOptions): Promise<vo
   if (!fs.existsSync(proxyIndexBrowserJsPath)) {
     await fs.promises.copyFile(path.join(__dirname, '../../index-browser.js'), proxyIndexBrowserJsPath)
   }
+
+  // we tell our vscode extension to reload the types by modifying this file
+  const prismaCache = paths('prisma').cache
+  const signalsPath = path.join(prismaCache, 'last-generate')
+  await fs.promises.mkdir(prismaCache, { recursive: true }).catch(() => {})
+  await fs.promises.writeFile(signalsPath, Date.now().toString()).catch(() => {})
```

Comment:
How does this work if you have multiple (different) projects open in VS Code? It would do a reload all of them?

Reply:
I think it might be not a big deal if it reloads all of them, but if we want to optimise that, we could write a path to the project instead of or together with the timestamp, and then let each language server instance decide if the notification is relevant for it.

Reply:
We decided that it could be a future optimisation